doc/lua_api.txt

```diff
@@ -5486,6 +5486,7 @@ Used by `ObjectRef` methods. Part of an Entity definition.
         --   of its texture.
         --   Otherwise for non-node items, the object will be an extrusion of
         --   'inventory_image'.
+		-- "node" mirrors behavior of "mesh" without the 'wield_image' check.
```

Comment:
Please use spaces to indent in the documentation.

Reply:
Whoops. Fixed.
I'll adjust the wording once a solid term is chosen.